src/src/WebServer/AdvancedConfigPage.cpp

```diff
@@ -191,7 +191,7 @@ void handle_advanced() {
 
 
   addFormSubHeader(F("Inter-ESPEasy Network"));
-
+  if (Settings.UDPPort != 8266 ) addFormNote(F("Default port is 8266"));
```

Comment:
This may be confusing.
Better state the "preferred" or "official" port to be 8266.